### ISSUE TITLE
[rllib] rllib doesn't work with tf.keras.layers.BatchNormalization

### ISSUE THREAD

**Issue by pimpke** [ray-project/ray#4831]
### Describe the problem

Below is a snippet of code from `__init__` of `tf_policy_graph` that gets the graph's UPDATE_OPS. The code assumes AssignMovingAvg ops of batch norm are contained inside the UPDATE_OPS collection:
```
# gather update ops for any batch norm layers
if update_ops:
    self._update_ops = update_ops
else:
    self._update_ops = tf.get_collection(
        tf.GraphKeys.UPDATE_OPS, scope=tf.get_variable_scope().name)
```
However, if `tf.keras.layers.BatchNormalization` is used inside the model, it won't add its update ops inside the collection. The way of fetching the updates ops from this keras layer should be done like in the code example below:

```
batch_norm = tf.keras.layers.BatchNormalization() 
output_tensor = batch_norm(input_tensor)
update_ops = batch_norm.updates  # this must be done after the layer has been called like in the previous line
```

**Comment by ericl** [comment]
What's a good solution to this? Should the keras layer be manually registered in some way or can the ops be added to the right graph keys?
